addons/osfstorage/tests/test_models.py

```diff
@@ -731,16 +736,16 @@ def test_fork_reverts_to_using_user_storage_default(self):
         child_settings.save()
 
         fork = project.fork_node(Auth(user))
-        child_fork = models.Node.objects.get_children(fork).first()
-        assert fork.get_addon('osfstorage').region_id == us.id
-        assert fork.get_addon('osfstorage').user_settings == user.get_addon('osfstorage')
-        assert child_fork.get_addon('osfstorage').region_id == us.id
+        fork_region_id = fork.get_addon('osfstorage').region_id
 
-        fork = project.fork_node(Auth(user2))
         child_fork = models.Node.objects.get_children(fork).first()
-        assert fork.get_addon('osfstorage').region_id == canada.id
-        assert fork.get_addon('osfstorage').user_settings == user2.get_addon('osfstorage')
-        assert child_fork.get_addon('osfstorage').region_id == canada.id
+        child_fork_region_id = child_fork.get_addon('osfstorage').region_id
+
+        assert fork_region_id == project.get_addon('osfstorage').region_id
```

Comment:
This assert (and the equivalent one below for the child_for_region_id) seem to be unnecessary, because the equivalency test is the same as the assignment with nothing happening in-between the assignment and the assertion. Unless you think the assignment is going to somehow fail quietly.